test-it/test.js

```diff
@@ -510,7 +516,7 @@
                         process.exit(errorCode);
                     });
                 }
-            });
+            }, execOnly, testsBaseDir);
```

Comment:
Here runNextTest is being invoked with four parameters (testFiles, a function, execOnly, testsBaseDir) but the runNextTest function only has two arguments (testFiles, chain) and does not seem to do anything fancy for retrieving these additional arguments. Are we sure this is behaving as expected?